src/NovaVaultV2.sol

```diff
@@ -5,17 +5,36 @@ import {LibSwap} from "@lifi/src/Libraries/LibSwap.sol";
 import {LibAllowList} from "lifi/Libraries/LibAllowList.sol";
 import {IERC20} from "./interfaces/IERC20.sol";
 import {IUniPool} from "./interfaces/IUniPool.sol";
+import {ReentrancyGuard} from "@solmate/utils/ReentrancyGuard.sol";
```

Comment:
@Imad-Allal can we import an "Ownable" contract from solmate as well?

Reply:
Yeah sure